website/docs/language/meta-arguments/lifecycle.mdx

```diff
@@ -49,6 +49,13 @@ The arguments available within a `lifecycle` block are `create_before_destroy`,
   such features, so you must understand the constraints for each resource
   type before using `create_before_destroy` with it.
 
+  Note that Terraform propagates and applies `create_before_destroy` meta-attribute
+  behaviour to all resource dependencies. For example, if resource A with enabled
+  `create_before_destroy` depends on resource B with disabled `create_before_destroy`
+  (by default), then Terraform enables `create_before_destroy` for resource B
+  implicitly and stores it to the state file. You cannot override `create_before_destroy`
+  to `false` on resource B, because that would imply dependency cycles in the graph.
```

Comment:
```suggestion
  behaviour to all resource dependencies. For example, if `create_before_destroy` is enabled on resource A but not on resource B, but resource A is dependent on resource B, then Terraform enables `create_before_destroy` for resource B 
  implicitly by default and stores it to the state file. You cannot override `create_before_destroy`
  to `false` on resource B because that would imply dependency cycles in the graph.
```
Just a few more tweaks for consistency with our style guidance.